guides/common/modules/proc_recovering-a-corrupted-repository.adoc

```diff
@@ -23,8 +23,9 @@ Use this option if you have one of the following errors:
 . In the {ProjectWebUI}, navigate to *Content* > *Products*.
 . Select the product containing the corrupted repository.
 . Select the name of a repository you want to synchronize.
-. From the *Select Action* menu, select *Advanced Sync*.
-. Select the option and click *Sync*.
+. To perform optimized sync or complete sync, select *Advanced Sync* from the *Select Action* menu.
+. Select the required option and click *Sync*.
+. To verify the checksum, click *Verify Content Checksum* from the *Select Action* menu. (optional)
```

Comment:
Actually one thing I hadn't seen. This should be like this: 
```suggestion
. Optional: To verify the checksum, click *Verify Content Checksum* from the *Select Action* menu. 
```

Reply:
Noted @melcorr 

I will take this minor change in the new guide i.e. v6.11 and will push it again.

Thank you.